Copy AC documentation to the AC module repository (#22)
kyma-project/application-connector-manager: docs/user/tutorials/01-30-register-secured-api.md

    displayName: "My Client Cert Service"
    description: "My service"
    providerDisplayName: "My organisation"
    entries:
    - credentials:
        secretName: {SECRET_NAME}
        type: CertificateGen
      targetUrl: {TARGET_API_URL}
      type: API
```

This is an example of the Secret containing credentials: 

```yaml
apiVersion: v1
kind: Secret
metadata:
  name: {SECRET_NAME}
  namespace: kyma-system
data:
  crt: {BASE64_ENCODED_CERTIFICATE}
  key: {BASE64_ENCODED_PRIVATE_KEY}
```

To create such a Secret, run this command:

```bash
kubectl create secret generic {SECRET_NAME} --from-literal crt={CERTIFICATE} --from-literal key={PRIVATE_KEY} -n kyma-system
```

## Register a CSRF-protected API

This is an example of the **service** object for an API secured with both Basic Authentication and a CSRF token:

```yaml
  - id: {TARGET_UUID} 
    name: my-csrf-service
    displayName: "My CSRF Service" 
    description: "My service"
    providerDisplayName: "My organisation"
    entries:
    - credentials:
        secretName: {SECRET_NAME}
        type: Basic
        csrfInfo:
          tokenEndpointURL: {CSRF_TOKEN_URL}
      targetUrl: {TARGET_API_URL}
      type: API
```

> > **NOTE:** It is assumed that the CSRF token endpoint service uses the same credentials as the target API.

Reviewer: ```suggestion
> **NOTE:** The example assumes that the CSRF token endpoint service uses the same credentials as the target API.
```
Do we assume that in the example or in general? I'd try to avoid the passive voice if possible
